aplib.net-demo/Assets/Scripts/Doors/TempItemID.cs

```diff
@@ -0,0 +1,9 @@
+using UnityEngine;
+
+/// <summary>
+/// Temporary class that gives an ID to the player, change the ID in the editor of the player object
+/// </summary>
+public class TempItemId : MonoBehaviour
+{
+    [SerializeField] public int itemId;
```

Comment:
If you add the tag "SerializeField" it will make this variable accessible within the unity editor. It is mainly used for private variables so it looks somewhat out of place here. Nonetheless it doesn't harm the code, but I'd recommend changing this.